.github/workflows/linux_conformance.yml

```diff
@@ -0,0 +1,942 @@
+name: Conformance Tests on Linux (Ubuntu 20.04, Python 3.11)
```

Comment:
I hope we will run conformance as a part of main Linux pipeline, right?

Reply:
Yes. I've excluded it into a separate workflow for testing purposes.